server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js

```diff
@@ -128,31 +131,32 @@ export default class CreationDateFacet extends React.PureComponent {
       return null;
     }
 
-    const data = periods.map((startDate, index) => {
-      const startMoment = moment(startDate);
-      const nextStartMoment =
-        index < periods.length - 1
-          ? moment(periods[index + 1])
-          : createdBefore ? moment(createdBefore) : undefined;
-      const endMoment = nextStartMoment && nextStartMoment.clone().subtract(1, 'days');
+    const { formatDate } = this.context.intl;
+    const data = periods.map((start, index) => {
+      const startDate = new Date(start);
+      const beforeDate = createdBefore ? createdBefore : undefined;
+      let nextStartDate = index < periods.length - 1 ? periods[index + 1] : beforeDate;
+      let endDate;
+      if (nextStartDate) {
+        nextStartDate = new Date(nextStartDate);
+        endDate = new Date(nextStartDate);
+        endDate.setDate(endDate.getDate() + 1);
+      }
 
       let tooltip =
-        formatMeasure(stats[startDate], 'SHORT_INT') + '<br>' + startMoment.format('LL');
-
-      if (endMoment) {
-        const isSameDay = endMoment.diff(startMoment, 'days') <= 1;
-        if (!isSameDay) {
-          tooltip += ' – ' + endMoment.format('LL');
-        }
+        formatMeasure(stats[start], 'SHORT_INT') +
+        '<br/>' +
+        formatDate(startDate, longFormatterOption);
+      if (endDate && !isSameDay(endDate, startDate)) {
+        tooltip += ' – ' + formatDate(endDate, longFormatterOption);
```

Comment:
when there are several years, the tooltips are like "January 1, 2011 – January 2, 2012", which is incorrect